src/rdb.h

```diff
@@ -73,12 +73,14 @@
 #define RDB_TYPE_STREAM_LISTPACKS_2 19
 #define RDB_TYPE_SET_LISTPACK  20
 #define RDB_TYPE_STREAM_LISTPACKS_3 21
-#define RDB_TYPE_HASH_METADATA 22
-#define RDB_TYPE_HASH_LISTPACK_EX 23
+#define RDB_TYPE_HASH_METADATA_PRE_GA 22      /* Hash with HFEs. Doesn't attach min TTL at start */
+#define RDB_TYPE_HASH_LISTPACK_EX_PRE_GA 23   /* Hash LP with HFEs. Doesn't attach min TTL at start */
```

Comment:
what about adding the version being used? like `#define RDB_TYPE_MODULE_PRE_GA 6 /* Used in 4.0 release candidates */`.

Reply:
we also need to update the `rdb_type_string` in redis-check-rdb.c